MPM/mpm_gateway.php

```diff
@@ -173,7 +173,15 @@ public function process_payment($order_id)
 			return array('result' => 'failure');
 		}
 
-		$order->update_status('pending', __('Awaiting payment confirmation', 'MPM'));
+		// We put a banktransfer order 'on-hold'. Woo will deduct stock and won't cancel the order after # minutes.
```

Comment:
Comment is niet helemaal correct maar dat maakt niet uit. Voorraad wordt pas bij `$order->payment_complete` afgeboekt. `update_status('on-hold')` doet niks anders dan status wijzigen.